mnd/mnd/templates/rdrf_cdes/index.html

```diff
@@ -28,7 +28,7 @@
   </div>
   <div class="links d-none d-md-block">
     <div>
-      <a href="{% url 'two_factor:login' %}?next=/router/"
+      <a href="{% url 'two_factor:login' %}?next=/mnd/"
```

Comment:
There are a few 'landing' pages to pick from when a user logs in:

1. A patient demographics page (in cases where the user is managing a single patient)
2. The 'patient listing page' (in cases where the user is allowed to see this)
3. The 'splash page' (if it exists)
4. The dashboard (if it exists)

I can't see a way to route this correctly generically for all registries, so I've just hard-coded it in mnd's settings & the login link.

One alternative would be to add an admin page that makes this more customisable?

Reply:
Could we make the default page customisable similar to how we override other functionality e.g. Registration Forms and Report Configurations?